### ISSUE TITLE
Rails/TimeZone cop should allow Time.parse against a fixed string which includes a timezone specifier.

### ISSUE THREAD

**Issue by urkle** [rubocop-hq/rubocop#9512]
## Is your feature request related to a problem? Please describe.

Often in our tests we have this construct
```ruby
Time.parse('2020-01-05T12:30:00Z')
```

Where we are specifically putting fixed value strings in the parse which ALWAYS have a timezone specifier (in this case it is Z meaning UTC)

## Describe the solution you'd like

The cop should allow this scenario (when using the full ISO8601 format with the timezone specifier)

## Describe alternatives you've considered

Simply disabling the cop fully for our specs.
